cli/schema/cypress.schema.json

```diff
@@ -98,24 +98,11 @@
           "default": "cypress/fixtures",
           "description": "Path to folder containing fixture files (Pass false to disable)"
         },
-        "integrationFolder": {
```

Comment:
Should `testFiles` also be deleted (or was that already deleted before this PR?)